src/components/Channels/ChannelsList.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import Accordion from "@mui/material/Accordion";
+import Box from "@mui/material/Box";
+import { Channel } from ".";
+import {
+  StyledAccordionDetails,
+  StyledAccordionExpandIcon,
+  StyledAccordionSummary,
+  StyledAccordionTitle
+} from "src/styles";
+
+interface IChannelsListProps {
+  listHeight: number;
+  channelList: string[];
+}
+
+const ChannelsList = ({ listHeight, channelList }: IChannelsListProps) => {
```

Comment:
why do you need to pass in an explicit height here in the component constructor? Is there a reason why the rendered height of the list must handled here and not by the styling?

Reply:
@telamonian so in every issue that is related to lists we have this part: _The box has a maximum height and overflow content scrolls. Expect this height to be a parameter_
I might have misunderstood this, but to me, it sounded like that there will be a use case where this will be managed through props. I could've simply put the max height in the styling but because of this line, I did things this way.

Reply:
That's... pretty weird. I think this probably is just a quirk of language and not a request for odd component constructor signatures. Since we want to reuse these components in at least a couple of very different styling and layolut contexts (standalone app, jupyterlab extension, etc) handling rendered this way seems burdensome.

As a general principle, we should assume component sizing is handled by the parent layout/styling and not by individual components themselves.

Reply:
@telamonian Understood and I agree, I'm removing all props that are related to this in all pull requests. We will handle list heights by parent layout styling or something similar. If I need to I'll add max height in styling directly for now.

Reply:
@telamonian Understood

Reply:
@ostojics  remove this `lineHeight` param, the idea was to avoid having a static height, styling with CSS later is fine/better.